app/src/ui/history/commit-list.tsx

```diff
@@ -499,7 +499,7 @@ export class CommitList extends React.Component<
         {this.renderReorderCommitsHint()}
         <List
           ariaLabel="Commits"
-          role={this.props.isInformationalView === true ? 'list' : 'list-box'}
```

Comment:
Yes.. Not only did I goof on checking for `undefined` against an item that couldn't be `undefined`... but also I yes switched `listbox` to `list-box` (my malapropism acting up..). But it most definitely is `listbox` https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role